AstSemantics.md

```diff
@@ -497,3 +515,10 @@ round-to-nearest ties-to-even rounding.
 Truncation from floating point to integer where IEEE-754 would specify an
 invalid operation exception (e.g. when the floating point value is NaN or
 outside the range which rounds to an integer in range) traps.
+
+Encoding a funcptr returns the index into the function table. If the index of
+the function is too great to fit in the result type, encoding traps. Decoding
+returns the funcptr from an encoded function index. If the index is out of
+bounds in the function table, decoding traps. In the MVP, funcptrs may only be
+convered to and from 32-bit integers. Support for 64-bit funcptrs may be added
+in the future.
```

Comment:
There are some scenarios where the table may not be densely packed, and one could imagine running out of 32-bit indices.